build/signedbuild.yml

```diff
@@ -133,13 +133,13 @@ jobs:
     inputs:
       inputType: Basic
       # Double-quote syntax with trailing backslash on each line concatenates lines without spaces
-      targets: "src\\AccessibilityInsights.Actions\\bin\\Debug\\Axe.Windows.*.dll;\
-                src\\AccessibilityInsights.Automation\\bin\\Debug\\Axe.Windows.*.dll;\
-                src\\AccessibilityInsights.Core\\bin\\Debug\\Axe.Windows.*.dll;\
-                src\\AccessibilityInsights.Desktop\\bin\\Debug\\Axe.Windows.*.dll;\
-                src\\AccessibilityInsights.Rules\\bin\\Debug\\Axe.Windows.*.dll\
-                src\\AccessibilityInsights.RuleSelection\\bin\\Debug\\Axe.Windows.*.dll;\
-                src\\AccessibilityInsights.Win32\\bin\\Debug\\Axe.Windows.*.dll;\
+      targets: "src\\Axe.Windows.Actions\\bin\\Debug\\Axe.Windows.*.dll;\
+                src\\Axe.Windows.Automation\\bin\\Debug\\Axe.Windows.*.dll;\
+                src\\Axe.Windows.Core\\bin\\Debug\\Axe.Windows.*.dll;\
+                src\\Axe.Windows.Desktop\\bin\\Debug\\Axe.Windows.*.dll;\
+                src\\Axe.Windows.Rules\\bin\\Debug\\Axe.Windows.*.dll\
+                src\\Axe.Windows.RuleSelection\\bin\\Debug\\Axe.Windows.*.dll;\
+                src\\Axe.Windows.Win32\\bin\\Debug\\Axe.Windows.*.dll;\
                 src\\Axe.Windows.Telemetry\\bin\\Debug\\Axe.Windows.*.dll;"
```

Comment:
These are intended to be alphabetically sorted, so Telemetry should come before Win32 (be sure to update the trailing backslashes)